src/components/bookList/BookList.tsx

```diff
@@ -0,0 +1,17 @@
+import BookCard from '../bookCard/BookCard'
+import './BookList.css'
+
+function BookList() {
+  return (
+    <div className='bookList'>
```

Comment:
Don't use camelCase for class names, use kebab-case. Additionally you can use this convention: https://getbem.com/